scripts/catalog_configmap.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+indent() {
+  INDENT="      "
+  sed "s/^/$INDENT/" | sed "s/^${INDENT}\($1\)/${INDENT:0:-2}- \1/"
+}
+
+CRDDIR=${DIR:-$(cd $(dirname "$0")/../deploy/crds && pwd)}
+PKGDIR=${DIR:-$(cd $(dirname "$0")/../deploy/olm-catalog/tektoncd-operator && pwd)}
+CSVDIR=${DIR:-$(cd ${PKGDIR}/0.0.1 && pwd)}
+
+NAME=${NAME:-tektoncd-operators}
+x=( $(echo $NAME | tr '-' ' ') )
+DISPLAYNAME=${DISPLAYNAME:=${x[*]^}}
+
+CRD=$(cat $(ls $CRDDIR/*crd.yaml) | grep -v -- "---" | indent apiVersion)
+CSV=$(cat $(ls $CSVDIR/*version.yaml) | indent apiVersion)
+PKG=$(cat $(ls $PKGDIR/*package.yaml) | indent packageName)
+
+cat <<EOF | sed 's/^  *$//'
+---
+apiVersion: v1
```

Comment:
we may want to add comment that this is autogenerated.

Reply:
added a comment inside the here document to indicate the resulting catalog manifest is autogenerated